### DISCUSSION TITLE
Workflows are saved to files, but webUI is not updated

### DISCUSSION THREAD

**Discussion by xer0times** [StackStorm/st2#6105] [UI]
I used the latest stable version of stackstorm, which is 3.8.1

It looks like after changing the workflow through the webui it will save the changes on disk (path `/opt/stackstorm/packs/TARGET/actions/workflow/TARGET.yaml`), but after opening the workflow again in the webui, I can't see the changes, I tried these commands, but no change was made:

```
st2ctl reload

st2-register-content

sudo st2ctl restart st2actionrunner st2api st2stream st2auth st2garbagecollector st2notifier st2rulesengine st2sensorcontainer st2chatops st2timersengine st2workflowengine st2scheduler st2web
```

**Comment by fdrab** [comment]
Ctrl + F5 works for me. The web page is probably cached.

**Reply by xer0times** [reply]
Sadly, it doesn't work for me!
